clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/deploy/description/servergroup/DeployKubernetesAtomicOperationDescription.groovy

```diff
@@ -82,6 +83,7 @@ class KubernetesContainerPort {
 
 @AutoClone
 @Canonical
+@Builder
```

Comment:
This sometimes requires both `@AllArgsConstructor` and `@NoArgsConstructor` for serialization & groovy magic to keep working

Reply:
It looks like the `@Canonical` is enough here; I'm able to create objects with a no args constructor an all args constructor as it was before the `@Builder` was added.